lib/methods/veres-one-client.js

```diff
@@ -212,7 +216,7 @@ class VeresOneClient {
    * Keys for signing the http request headers
    * @param [options.authKey] {LDKeyPair}
    *
-   * @returns response {Promise<Response>} from a fetch() POST.
+   * @returns response {Promise<Response>} from an axios POST request
```

Comment:
is this correct syntax.  It is notably different from what is on L131.

Reply:
I think L131 is just a more generic version of 219, I just put `object` instead of `Response` instance. I'll change it to match, just so there's no confusion (and in case I'm wrong about the instance name.)